{{ cookiecutter.package_name }}/setup.py

```diff
@@ -67,6 +63,12 @@
     package = sys.modules[PACKAGENAME]
     LONG_DESCRIPTION = package.__doc__
 
+# Enforce Python version check - this is the same check as in __init__.py but
+# this one has to happen before importing ah_bootstrap.
+if sys.version_info < tuple((int(val) for val in __minimum_python_version__.split('.'))):
```

Comment:
@Cadair , I thought this needs to happen *before* `import ah_bootstrap`. Has that changed?